src/components/Status/Status.styles.ts

```diff
@@ -0,0 +1,196 @@
+import { StatusAppearance } from './StatusAppearance.type';
+
+import { BaseProps } from '@/types';
+
+export type StatusConfig = {
+  appearance?: Partial<
+    Record<
+      StatusAppearance,
+      { emphasis?: Partial<Record<'high' | 'medium' | 'low', BaseProps>> }
+    >
+  >;
+  dot: {
+    appearance: Partial<
+      Record<
+        StatusAppearance,
+        { emphasis?: Partial<Record<'high' | 'medium' | 'low', BaseProps>> }
+      >
+    >;
+  } & BaseProps<'appearance'>;
+  hasLabel?: BaseProps;
+  innerElements: {
+    label?: BaseProps;
+  };
+} & BaseProps<'appearance'>;
+
+export const defaultConfig = {
+  display: 'flex',
+  w: 'fit-content',
+  h: '$size-xSmall',
+  gap: '$space-component-gap-small',
+  p: '$space-component-padding-null $space-component-padding-medium',
+  borderRadius: '$border-radius-medium',
+  alignItems: 'center',
+  text: '$typo-body-medium',
+  dot: {
```

Comment:
You should use `StatusDot` her with `custom` prop: http://localhost:6006/?path=/docs/global-primitives-statusdot--docs